utils/githooks/pre-commit.d/50-clang-format.sh

```diff
@@ -13,11 +14,11 @@ if [ -e .git/MERGE_HEAD ]; then
 fi
 
 if ! command -v git-clang-format > /dev/null 2>&1; then
-    echo "git-clang-format not installed. Skipping"
+    echo "git-clang-format not installed. See ./utils/githooks/README.md for instructions"
     exit 0
```

Comment:
`exit 1`
Shall we also make `git-clang-format` not optional?

Reply:
No objection from me. But TBH I don't write enough C code to know whether that will bother people

Reply:
If we enable this in CI we should also enable it in githooks.
Otherwise build fails and user will have to make another commit to fix that issue.

Reply:
If we get acceptance here we can also modify all other pre-commit scripts later to force everybody with pre-validation for every changes that are later verified by CI ;).

Reply:
> in the past people have not liked when a githook prevents them from committing

Funny thing clang changes over time and differs from distro to distro. So, you may even have clang-format pre-hook enabled and still be stopped by a check on GitHub. Just because of the different versions I believe.

Here `--no-verify` toggle becomes very useful. Just turn it off when it is in your way. ;-)